src/model/PlanarSurface_Impl.hpp

```diff
@@ -177,6 +178,8 @@ namespace detail {
 
     std::vector<GeneratorPhotovoltaic> generatorPhotovoltaics() const;
 
+    std::vector<SurfaceControlMovableInsulation> surfaceControlMovableInsulation() const;
+
     std::vector<SurfacePropertyConvectionCoefficients> surfacePropertyConvectionCoefficients() const;
```

Comment:
More of a side note at this point (didn't think about it for too long), but why are these two different:
* `std::vector<SurfaceControlMovableInsulation> PlanarSurface::surfaceControlMovableInsulation() const`
* `boost::optional<SurfaceControlMovableInsulation> Surface_Impl::surfaceControlMovableInsulation()`

And why isn't the latter calling the base class vector one if it makes sense?

Reply:
This is true for `surfacePropertyConvectionCoefficients` too right? I'm not sure why the two are different. I just used `surfacePropertyConvectionCoefficients` as a guide...

Reply:
Yeah true for `surfacePropertyConvectionCoefficients()` as well. 

I would assume that one was probably copying `generatorPhotovoltaics()` a couple of lines above in `PlanarSurface`.

Except one object is called `Generator:Photovoltaic` and generatorPhotovoltaics is supposed to really return an array. The other is called `SurfaceProperty:ConvectionCoefficients` (plural) and it's should have returned a `boost::optional<SurfacePropertyConvectionCoefficients> `object

Reply:
An alternative is that an earlier version might have returned both SurfacePropertyConvectionCoefficients and SurfacePropertyConvectionCoefficientsMultipleSurface

https://github.com/NREL/OpenStudio/blob/8ae4d883ed1a6ba704aa8b60003646de69a00f39/src/model/PlanarSurface.hpp#L235-L236

Anyways, I don't think it's right.